src/clusters.ts

```diff
@@ -11,16 +11,22 @@ export class ClusterVersion {
   version: string;
 }
 
-export async function createCluster(vendorPortalApi: VendorPortalApi, clusterName: string, k8sDistribution: string, k8sVersion: string, clusterTTL: string): Promise<Cluster> {
+export async function createCluster(vendorPortalApi: VendorPortalApi, clusterName: string, k8sDistribution: string, k8sVersion: string, clusterTTL: string, diskGib?: number, nodeCount?: number, instanceType?: string): Promise<Cluster> {
     const http = await vendorPortalApi.client();
 
     const reqBody = {
         "name": clusterName,
         "kubernetes_distribution": k8sDistribution,
         "kubernetes_version": k8sVersion,
         "ttl": clusterTTL,
-        "disk_gib": 50,
+        "disk_gib": diskGib || 50,
+        "node_count": nodeCount || 1,
     }
+
+    if (instanceType !== undefined) {
```

Comment:
should / can this just be in the initialization of the struct above, following the same pattern?

```"instance_type": instanceType || undefined```

? My TS might be way off here though, so this isn't a blocker.

Reply:
Yeah that would probably align better with what replicated CLI is doing as well (passing empty value)